src/pages/signup.tsx

```diff
@@ -67,10 +66,10 @@ const SignUpPage = () => {
               <img className="h-10 w-10 ml-6" src="/static/GoogleLogo.svg" />
```

Comment:
We can take care of this later as well if not already, but the 160 pixel density requirement by google on their logo- was that calculated yet?

Reply:
Oh good point--this should probably be addressed before we merge

Reply:
Raised in #24 

Let's address this in a separate PR